### DISCUSSION TITLE
note: module requires Go 1.19

### DISCUSSION THREAD

**Discussion by knowthetech** [projectdiscovery/nuclei#3324] [Q&A] [answered]
# github.com/projectdiscovery/utils/maps
../pkg/mod/github.com/projectdiscovery/utils@v0.0.9/maps/synclock_map.go:15:18: undefined: atomic.Bool
note: module requires Go 1.19
# github.com/projectdiscovery/nuclei/v2/pkg/protocols/common/hosterrorscache
../pkg/mod/github.com/projectdiscovery/nuclei/v2@v2.8.9/pkg/protocols/common/hosterrorscache/hosterrorscache.go:36:16: undefined: atomic.Int32
../pkg/mod/github.com/projectdiscovery/nuclei/v2@v2.8.9/pkg/protocols/common/hosterrorscache/hosterrorscache.go:116:40: undefined: atomic.Int32
note: module requires Go 1.19
# github.com/projectdiscovery/clistats
../pkg/mod/github.com/projectdiscovery/clistats@v0.0.12/clistats.go:86:30: undefined: atomic.Uint64
../pkg/mod/github.com/projectdiscovery/clistats@v0.0.12/clistats.go:117:37: undefined: atomic.Uint64
../pkg/mod/github.com/projectdiscovery/clistats@v0.0.12/fields.go:10:23: undefined: atomic.Uint64
note: module requires Go 1.19

**Comment by ehsandeep** [comment, marked as answer]
@knowthetech as the error says - `note: module requires Go 1.19`, You need to have go installed => go 1.19
